common/faceting.js

```diff
@@ -924,8 +945,15 @@
 
 
                 var appliedLen = scope.facetModel.appliedFilters.length;
+                var pageSize = tableConstants.PAGE_SIZE;
+                if (!scope.facetColumn.hideNullChoice) {
```

Comment:
Could you add a comment as to why the page size should be increased because of this filter. I was thinking the presence of this filter shouldn't change the page size used in the read function..

Reply:
This should be refactored.